js/util/evented.js

```diff
@@ -78,7 +78,14 @@ var Evented = {
      * @returns {Object} `this`
      */
     fire: function(type, data) {
-        if (!this.listens(type)) return this;
+        if (!this.listens(type)) {
+            // Error events are important for debugging and should not be
+            // silently dropped.
+            if (util.endsWith(type, 'error')) {
```

Comment:
What's your concern with checking the string @mourner? I believe there are cases where we fire legitimate error events for which `data && data.error instanceof Error` would be `false`.